source/api_documentation/transactions_api/_rollup_message.rst

```diff
@@ -0,0 +1,13 @@
+.. raw:: html
+
+  <h2>
+    Signal Transactions Rollup
+  </h2>
+  <div class="alert alert-danger">
+    <p>
+      Starting with version <b>2019-12-01</b>, all future API versions will only return rollup rows of transaction types "Call", "Sale", and "Post Call Event".
+    </p>
+    <p>
+      For more information, view the support documentation for <a href="https://invoca.force.com/community/s/article/How-to-access-Invoca-call-data-programmatically-via-API" target="_blank">best practices in accessing Invoca Call Data via the Transactions API</a>.
```

Comment:
Minor thing - the API documentation linked says new version / old version when describing post call events.  Can we put an actual version number in there once this has shipped.?

Reply:
Noted, I'll sync with Nash on that.